contributing_to_docs/doc_guidelines.adoc

```diff
@@ -287,98 +287,137 @@ Do not use backticks or other markup in assembly or module headings.
 ----
 [id="deployment-scaling-benefits_{context}"]
 == Deployment and scaling benefits
 ----
 
 === Anchoring "Prerequisites", "Additional resources", and "Next steps" titles in assemblies
 
-Use unique IDs for "Prerequisites", "Additional resources", and "Next steps" titles in assemblies. You can add the prefixes `prerequisites_`, `additional-resources_`, or `next-steps_` to a unique string that describes the assembly topic. The unique string can match the value assigned to the `:context:` attribute in the assembly.
+Use unique IDs for "Prerequisites", "Additional resources", and "Next steps" titles in assemblies. Add the prefixes `prerequisites_`, `additional-resources_`, or `next-steps_` to a unique string that describes the assembly topic.
 
 [NOTE]
 ====
 The `prerequisites_`, `additional-resources_`, and `next-steps_` prefixes must end with an underscore (`_`) when declared in an anchor ID in an assembly.
 ====
 
-The following examples include IDs that are unique to the "Configuring alert notifications" assembly:
+You can use the `{context}` variable to automatically match the value assigned to the `:context:` attribute in the assembly. Alternatively, you manually set the string after the `_`, provided that the full string that you form is unique across the repo.
 
-*Example unique ID for a "Prerequisites" title*
+The following examples include IDs that are unique to a "Configuring alert notifications" assembly that has `:context: configuring-alert-notifications` set in its metadata:
 
+.Example unique ID for a "Prerequisites" title
 ----
-[id="prerequisites_configuring-alert-notifications"]
+[id="prerequisites_{context}"]
 == Prerequisites
 ----
 
-*Example unique ID for an "Additional resources" title*
+.Alternative example with manually set string
+----
+[id="prerequisites_configuring-alert-notifications"]
+== Prerequisites
+----
 
+.Example unique ID for an "Additional resources" title
 ----
 [role="_additional-resources"]
-[id="additional-resources_configuring-alert-notifications"]
+[id="additional-resources_{context"]
 == Additional resources
 ----
 
-*Example unique ID for a "Next steps" title*
-
+.Example unique ID for a "Next steps" title
 ----
-[id="next-steps_configuring-alert-notifications"]
+[id="next-steps_{context"]
 == Next steps
 ----
 
+=== Discrete headings
+
+If you have a section heading that you do not want to appear in the TOC (for example, if you think that some section is not worth showing up or if there are already too many nested levels), you can use a discrete (or floating) heading:
+
+https://docs.asciidoctor.org/asciidoc/latest/blocks/discrete-headings/
+
+A discrete heading also will not get a section number in the Customer Portal build of the doc. Previously, we would use plain bold mark-up around a heading like this, but discrete headings also allow you to ignore section nesting rules (like jumping from a `==` section level to a `====` level if you wanted for some style reason).
```

Comment:
Hmm probably not, hadn't considered. Tho now that I re-read, I do kind of like that it discourages "plain bold mark-up" as a fake heading, in case someone thought of that on their own later.